model/team.go

```diff
@@ -97,7 +97,7 @@ func (o *Team) Etag() string {
 	return Etag(o.Id, o.UpdateAt)
 }
 
-func (o *Team) IsValid() *AppError {
```

Comment:
We can't import `utils` into a model since that would introduce a cyclic dependency